pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -590,6 +590,96 @@ roleRef:
 	require.Equal(t, expectedOutput, output)
 }
 
+func TestChangeGraphWithAppCR_SA_RoleAndRoleBinding(t *testing.T) {
```

Comment:
Thought of an even better name than this (with only one `_`), but feel free to use something else
```suggestion
func TestChangeGraphWithAppCR_RoleRoleBindingAndSA(t *testing.T) {
```

Reply:
How about  `TestChangeGraphWithAppCR_RBACAndSA`  or  `TestChangeGraphWithAppCR_RBACAndSvcAccount` ?

Reply:
> How about TestChangeGraphWithAppCR_RBACAndSA or TestChangeGraphWithAppCR_RBACAndSvcAccount ?

`...RBACAnd...` it's hard to read when an acronym is followed by `And`, plus not a very big fan of `SvcAccount`